test/common/testlib.py

```diff
@@ -1600,6 +1600,7 @@ def terminate_sessions():
 
             # Restart logind to mop up empty "closing" sessions
             self.machine.execute("systemctl restart systemd-logind")
+            self.machine.execute("systemctl restart systemd-user-sessions.service")
```

Comment:
Not sure if that actually does any good, but as long as tests are green, I'm happy :grin: